master/buildbot/steps/package/deb/pbuilder.py

```diff
@@ -77,6 +79,10 @@ def __init__(self,
             self.distribution = distribution
         if mirror:
             self.mirror = mirror
+        if othermirror:
+            if type(othermirror) is list:
```

Comment:
Let's require it to be a list. This would be one less special case to support and also lead to a slightly more consistent interface.

Reply:
Sounds good, will do. And now I understand why the coverage went down.